src/Compatibility/Microsoft.DotNet.ApiCompatibility/Abstractions/Mappers/ElementSide.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.ApiCompatibility.Abstractions
+{
+    /// <summary>
+    /// Enum representing the side of an element in the mappers.
+    /// </summary>
+    public enum ElementSide : byte
```

Comment:
curious why we are inheriting the byte class ?

Reply:
Because `ElementSide` will never contain more than 2 values, so no need to use the default `int` underlying type for the enum fields, we can just use byte to use smaller fields memory wise.